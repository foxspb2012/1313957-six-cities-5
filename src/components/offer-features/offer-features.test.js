import React from 'react';
import renderer from 'react-test-renderer';
import OfferFeatures from './offer-features';

describe(`Should OfferFeatures render correctly`, () => {
  it(`Should OfferFeatures render correctly with bedrooms and adults`, () => {
    const tree = renderer
      .create(
          <OfferFeatures
            type={`apartment`}
            bedroomAmount={3}
            guestAmount={4}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`Should OfferFeatures render correctly without bedrooms and adults`, () => {
    const tree = renderer
      .create(
          <OfferFeatures type={`apartment`}/>
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
