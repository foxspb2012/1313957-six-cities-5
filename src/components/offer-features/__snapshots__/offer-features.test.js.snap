// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OfferFeatures render correctly Should OfferFeatures render correctly with bedrooms and adults 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Apartment
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
    3
     Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
    4
     adults
  </li>
</ul>
`;

exports[`Should OfferFeatures render correctly Should OfferFeatures render correctly without bedrooms and adults 1`] = `
<ul
  className="property__features"
>
  <li
    className="property__feature property__feature--entire"
  >
    Apartment
  </li>
  <li
    className="property__feature property__feature--bedrooms"
  >
     Bedrooms
  </li>
  <li
    className="property__feature property__feature--adults"
  >
    Max 
     adults
  </li>
</ul>
`;
